Tests/Realm.Tests/Sync/StaticQueriesTests.cs

```diff
@@ -1257,7 +1257,7 @@ private async Task<MongoClient.Collection<T>> GetCollection<T>(string appConfigT
 
             SyncConfigurationBase config = appConfigType == AppConfigType.FlexibleSync ? GetFLXIntegrationConfig(user) : GetIntegrationConfig(user);
 
-            using var realm = await GetRealmAsync(config);
```

Comment:
We need to wait for the schema to be synchronized, otherwise the call to the backends without the database name could raise an exception